### PR TITLE
Fix the centos7-hammer-devel box to `become` less

### PR DESCRIPTION
**Pull request by akofink**
Related to https://github.com/theforeman/forklift/pull/627 and https://github.com/theforeman/forklift/pull/668.

We want to install a hammer_devel environment in the `"{{ remote_user }}"`'s home directory, not root's.

### CODE REVIEW

File: playbooks/hammer_devel.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
 - hosts: all
-  become: yes
   roles:
     - etc_hosts
     - git
-    - ruby_scl
+    - { role: ruby_scl, become: yes }
```
Comment (ehelms) [inline playbooks/hammer_devel.yml]: Can this be:

```
- role: ruby_scl
  become: yes
```

I find the hash syntax a bit jarring in yaml.

Comment (akofink) [reply playbooks/hammer_devel.yml]: Yes. Yes it can. :)

**Comment by akofink** [comment]
Updated

**Comment by akofink** [comment]
Would love to get this in so centos7-hammer-devel isn't broken. :)

**Comment by ekohl** [comment]
Would it be a best practice to use ```become: true``` in the role instead? Here my ansible knowledge is limited.

**Comment by akofink** [comment]
@ekohl That's how it was originally implemented until @ehelms changed it. Perhaps @ehelms could explain his reasoning and offer an opinion?

**Comment by akofink** [comment]
Unless removing the `become` was a mistake, in which case, I'm not opposed to re-adding it on the role itself. @ehelms?

**Pull request merged**